src/hooks/useCroct.ssr.test.tsx

```diff
@@ -1,12 +1,12 @@
 import {renderHook} from '@testing-library/react-hooks/server';
-import {ReactNode} from 'react';
+import {ReactElement} from 'react';
 import {useCroct} from './useCroct';
 import {CroctProvider} from '../CroctProvider';
 
 describe('useCroct', () => {
     it('should not fail on server-side rendering', () => {
         const {result} = renderHook(() => useCroct(), {
-            wrapper: ({children}: {children?: ReactNode}) => (
+            wrapper: ({children}: {children?: ReactElement}) => (
```

Comment:
is `{children?: ReactElement}` still needed?

Reply:
Yes, because `children` cannot be `undefined`